package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
```

Comment:
Since we now use ES6, would it be better to set `"format": "es6"` and *not* use the `dist` JavaScript?